src/index.ts

```diff
@@ -134,17 +140,26 @@ export class I2C extends Peripheral {
     super.destroy();
   }
 
-  private getDevice(address: number) {
+  private getDevice(address: number, busNumber?: number) {
     let device = this.devices[address];
 
     if (device === undefined) {
-      device = openSync(getBoardRevision() === VERSION_1_MODEL_B_REV_1 ? 0 : 1);
+      if (busNumber === undefined) {
+        busNumber = this.busNumber;
+      }
+      device = openSync(busNumber);
       this.devices[address] = device;
     }
 
     return device;
   }
 
+  public configure(options?: any): void {
```

Comment:
Here's what the IO-Plugins documentation has to say about specifying which I2C bus a device is connected to: https://github.com/rwaldron/io-plugins#i2cconfigoptions

A `bus` may be specified for an I2C device by a user program when the corresponding component is instantiated and the `bus` specified will be forwarded to `i2cConfig`. Johnny-Five doesn't really care what bus a device is connected to and it's up to the IO-Plugin to decide what to do with any `bus` specified. In reality, this means that Johnny-Five can support more than one bus.